hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationServiceBuilder.java

```diff
@@ -63,5 +64,7 @@
 
     SerializationServiceBuilder setInitialOutputBufferSize(int initialOutputBufferSize);
 
+    SerializationServiceBuilder setClusterService(ClusterService clusterService);
```

Comment:
yeah, I know, it's not perfect. I need a clusterVersion on the write/read call.
we can also change the signatures, but it's used sometimes only